src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -108,4 +122,116 @@ public void exec(final URI home) throws IOException {
             }
         );
     }
+    
+    /**
+     * Try to send 2 requests in 1 tcp connection.
+     * @throws IOException
+     */
+    @Test
+    public void persistenceConnection() throws IOException {
```

Comment:
@dmzaytsev 
There are actually two requests.
The last assert statement checks whether two "Hello world" have replied by server.
Maybe it is a bit hard to read, I will refactor it.